src/components/Dropdown.tsx

```diff
@@ -0,0 +1,46 @@
+import { Dispatch, SetStateAction } from 'react';
+
+/**
+ * Dropdown Component
+ * 
+ * This page displays a simple dropdown that shows the passed in string array elements as select options.
+ */
+interface DropdownProps {
+    title: string;
+    options: string[];
+    selectedIndex: number;
+    setSelectedIndex: Dispatch<SetStateAction<number>>;
+}
+
+const Dropdown = ({
+    title,
+    options,
+    selectedIndex,
+    setSelectedIndex
+}: DropdownProps) => {
+    const handleChange = (item: number) => {
+        setSelectedIndex(item)
+    };
+
+    return (
+        <>
+            {title}
+            <div>
+                <select defaultValue={options[selectedIndex]}>
+                    {/* <option value="none">None</option> */}
```

Comment:
We can use this option once we update the app to _not_ have a default value on load.